src/WatchDogs.Domain/SuspiciousDealDetectorOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WatchDogs.Domain;
+
+/// <summary>
+/// There are different types of lot sizes, which allow traders to control how large their positions are.
+/// Here we are storing them because we need them in calculating volume-to-balance ratio.
+/// Because this app should be scaleable we provide all standardized lot types in financial markets, particularly in forex trading
+/// </summary>
+public record SuspiciousDealDetectorOptions
+{
+    public int StandardLot { get; set; }
+    public int MiniLot { get; set; }
```

Comment:
Why can't this be simply expressed as decimal?

Reply:
Standard Lot: 100,000 units of the base currency.
Mini Lot: 10,000 units.
Micro Lot: 1,000 units.
Nano Lot: 100 units (though this is less common).
Official numbers from Forex Market